docs/quickstart.md

```diff
@@ -108,13 +109,14 @@ npm install --save firebase reactfire
    //...
    ```
 
-## 6. Modify `src/App.js`
+## 6. Add the Burrito() function to `App.js`
```

Comment:
please use "function component" instead of "function" to match the [React team's terminology](https://reactjs.org/docs/components-and-props.html#function-and-class-components)